netbout-client/src/main/java/com/netbout/client/RtAlias.java

```diff
@@ -108,11 +110,24 @@ public void photo(final URI uri) {
 
     @Override
     public String email() throws IOException {
-        return this.request.fetch()
-            .as(XmlResponse.class)
-            .xml()
-            .xpath("/page/alias/email/text()")
-            .get(0);
+        final XML body = this.request.fetch().as(XmlResponse.class).xml();
+        final List<String> lverified = body
```

Comment:
That would mean calling twice `body.xpath("/page/alias/email/text()")`. And that would mean:

1. Searching the XML twice for the same tag.
2. We need to extract the path `"/page/alias/email/text()"` to a constant, since we cannot use a String twice according to Qulice.

And then, why do you consider extracting `alias.emal()` to a variable but prefer to inline `body.xpath()`?